src/api.rs

```diff
@@ -76,8 +76,15 @@ pub struct EncoderConfig {
   pub quantizer: usize,
   pub bitrate: i32,
   pub tune: Tune,
+  /// log2(tile columns). If tiles is also specified, this acts
+  /// as a minimum.
   pub tile_cols_log2: usize,
+  /// log2(tile rows). If tiles is also specified, this acts
+  /// as a minimum.
```

Comment:
IMO, either the user should provide `--tile-cols-log2` and `--tile-rows-log2, or `--tiles`, but never both.

Reply:
OK. We can error out in this case, or just make one completely override the other.